include/jwt-cpp/jwt.h

```diff
@@ -1570,6 +1570,18 @@ namespace jwt {
 					throw error::ecdsa_exception(error::ecdsa_error::invalid_key_size);
 			}
 
+			ecdsa(helper::evp_pkey_handle private_key, const EVP_MD* (*md)(), std::string name, size_t siglen)
```

Comment:
I currently only added the minimal code required to make my use case work. Please let me know if passing an EVP_PKEY directly is something that this library is willing to support in the future.
If yes, we can think about how to fully implement this feature. I would probably also need to add an rsa implementation...